flytekit/extras/accelerators.py

```diff
@@ -1,3 +1,93 @@
+"""
+Use specific accelerators
+==========================
+
+.. tags:: MachineLearning, Advanced, Hardware
+
+Flyte allows you to specify `gpu` resources for a given task. However, in some cases, you may want to use a different
+accelerator type, such as TPU, specific variations of GPUs, or fractional GPUs. You can configure the Flyte backend to
+use your preferred accelerators, and those who write workflow code can import the `flytekit.extras.accelerators` module
+to specify an accelerator in the task decorator.
+
+
+If you want to use a specific GPU device, you can pass the device name directly to the task decorator, e.g.:
+
+.. code-block::
+
+    @task(
+        limits=Resources(gpu="1"),
+        accelerator=GPUAccelerator("nvidia-tesla-v100"),
+    )
+    def my_task() -> None:
+        ...
+
+
+Base Classes
+------------
+These classes can be used to create custom accelerator type constants. For example, you can create a TPU accelerator.
+
+
+
+.. currentmodule:: flytekit.extras.accelerators
+
+.. autosummary::
```

Comment:
Add toctree option to generate doc for each class, see [here](https://flyte--2076.org.readthedocs.build/projects/flytekit/en/2076/generated/flytekit.deck.Deck.html) for example.

```suggestion
.. autosummary::
   :toctree: generated/
```

Reply:
i dont like it, as it would separate each one into a separate file, but they are actually just a table?